workspaces/tech-radar/plugins/tech-radar/overlay/package.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Ideally, we would like to use a `patch` file at the workspace level in the overlay, instead of a overriding with a complete `package.json`.